pyproject.toml

```diff
@@ -33,7 +33,16 @@ classifiers = [
 python = "^3.7"
 SQLAlchemy = ">=1.4.36,<2.0.0"
 pydantic = "^1.9.0"
-sqlalchemy2-stubs = {version = "*", allow-prereleases = true}
+sqlalchemy2-stubs = { version = "*", allow-prereleases = true }
+typer = "^0.9.0"
+alembic = "^1.12.0"
```

Comment:
I feel that adding a dependency needs a formal discussion.

On one hand, `alembic` is pretty lean, seems to only depend on `sqlalchemy` which this package already depends on too.

On the other, there are bound to be users that don't generate or apply migrations, rather consume some existing database instead. These users get a bit of bloat, and possibly a restriction on sqlalchemy versions they can use in their application.

This specific library addition may have been carefully thought through and may be OK.

A random library addition would be a problem.

Thus, I'd love to see a discussion and maintainer vote/rationale.

Reply:
As I mentioned above, we used alembic happily with sqlmodel, so maybe it's better indeed for sqlmodel to not depend on it, but just have things so that alembic can be used for it .. like it already can be.

Reply:
Any updates about this?

Reply:
Maybe it should be a separate package, just like alembic is separated from sqlalchemy. That would also allow this to be released more easily and have people play with it, test it. Have you thought about releasing it @Jufik ?